python/GafferArnoldTest/IECoreArnoldPreviewTest/RendererTest.py

```diff
@@ -2670,6 +2670,48 @@ def testCameraAttributes( self ) :
 			uvRemapNode =  arnold.AiNodeGetLink( cameraNode, "uv_remap" )
 			self.assertEqual( arnold.AiNodeEntryGetName( arnold.AiNodeGetNodeEntry( uvRemapNode ) ), "flat" )
 
+	def testCantEditQuadLightColor( self ) :
```

Comment:
I was anticipating a test that we _can_ edit quad light colour, using InteractiveArnoldRender and showing that edits we make are reflected in the rendered image. I'm looking for a test that will still be valuable once the Arnold bug is fixed.